ui/app/templates/components/clients/current.hbs

```diff
@@ -19,7 +19,16 @@
         FILTERS
         <Toolbar>
           <ToolbarFilters>
-            {{! ARG TODO more filters for namespace here }}
+            <SearchSelect
+              @id="namespace-search-select-monthly"
+              @models={{this.searchSelectModel}}
+              @selectLimit="1"
+              @disallowNewItems={{true}}
+              @fallbackComponent="input-search"
+              @onChange={{action "selectNamespace"}}
```

Comment:
Is the most updated way of doing this `{{action this.selectNamespace}}`

Reply:
I'll ask Jordan, but I've never figured out how to get this to work in glimmer using the 'on' modifier. Specifically, passing an action to a component.

Reply:
Got it! Don't need the action handler it appears at all.

Reply:
So wait, this can just be `@onChange={{this.selectNamespace}}` ?

Reply:
it is. I didn't realize that. I should be this in the updated file.